### PR TITLE
Release notes 1.3.0

### PR DESCRIPTION
**Pull request by hubertmis**

### CODE REVIEW

**Comment by hubertmis** [comment]
@krlo , backwards compatibility is provided automatically. The driver checks run-time version of installed SoftDevice during initialization and the driver configures itself. 
Should I add this information to the release notes? I assumed if I don't state anything about backward compatibility it means it is automatically preserved.

File: release_notes/nrf_802154_1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# nRF IEEE 802.15.4 radio driver 1.3.0 release notes
+
+Release date: 2019-02-04
+
+This release contains a support for a new chip: nRF52811. It also adds an Enhanced Acks parser and generator. The release also includes fixes to bugs found since the last release.
+
+## New features:
+* Created a generator of Enahnced Acks and improved parsing of IEEE 802.15.4-2015 frames. ([PR #179](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/pull/179))
```
Comment (b-gent) [inline release_notes/nrf_802154_1.3.0.md]:
``` suggestion
* Added a generator of Enhanced Acks and improved parsing of IEEE 802.15.4-2015 frames. ([PR #179](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/pull/179))
```

File: release_notes/nrf_802154_1.3.0.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* Provided support for new chip: nRF52811. ([PR #190](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/pull/190))
```
Comment (b-gent) [inline release_notes/nrf_802154_1.3.0.md]:
``` suggestion
* Provided support for a new chip: nRF52811. ([PR #190](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/pull/190))
```

File: release_notes/nrf_802154_1.3.0.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+* Added a configuration option to delay the CCA operations during the CSMA-CA procedure until a timeslot is granted by an aribter. ([PR #191](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/pull/191))
```
Comment (b-gent) [inline release_notes/nrf_802154_1.3.0.md]:
``` suggestion 
* Added a configuration option to delay CCA operations during the CSMA-CA procedure until a timeslot is granted by an arbiter. ([PR #191](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/pull/191))
```

File: release_notes/nrf_802154_1.3.0.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## Bug fixes:
+* Timeslots are actively released when the higher layer request transition to the sleep state. Passive releasing of timeslots was a source of failed operations. Operations were failing when the higher layer requested an operation short time (less than 15 ms) after it requested transition to the sleep state. ([PR #105](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/pull/105))
```
Comment (b-gent) [inline release_notes/nrf_802154_1.3.0.md]:
``` suggestion
* Timeslots are actively released when the higher layer requests a transition to the sleep state. Passive releasing of timeslots was found to be a source of failed operations. Operations were failing when the higher layer requested an operation a short time (less than 15 ms) after it requested a transition to the sleep state. ([PR #105](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/pull/105))
```

**Pull request merged**